config.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020 Chair of Applied Cryptography, Technische Universität
+// Darmstadt, Germany. All rights reserved. This file is part of
+// perun-eth-demo. Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package prnm
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sirupsen/logrus"
+	"perun.network/go-perun/apps/payment"
+	"perun.network/go-perun/backend/ethereum/wallet"
+
+	"perun.network/go-perun/log"
+	plogrus "perun.network/go-perun/log/logrus"
+)
+
+// Config complete configuration needed to operate the Client.
+type Config struct {
+	// Name to be used in state channels.
+	Alias string
+	// Address will be unlocked in the KeyStore with password
+	KeyStorePath, Password string
+	Address                *Address // OnChain address also calles PerunID.
+	DatabasePath           string   // Path to the database file.
+	ETHNodeURL             string   // URL of the ETH node. Example: ws://127.0.0.1:8545
+	IP                     string   // Ip to listen on.
+	Port                   uint16   // Port to listen on.
+	// TODO loglevel
```

Comment:
I think we can remove this TODO, as the loglevel is not a client-level configuration, but framework-globally.